docs/Makefile

```
@@ -6,6 +6,15 @@ CONTAINER_IMAGE  ?= cilium/tetragon-hugo:v$(HUGO_VERSION)
 CONTAINER_RUN    ?= "$(CONTAINER_ENGINE)" run --rm --interactive --tty --volume "$(abspath $(CURDIR)/..):/src" --workdir /src/docs
 HUGO_DOCKERFILE  ?= Dockerfile.hugo
 
+# Tracing Policy docs generation settings
+TRACING_POLICY_TEMPLATE := content/en/docs/reference/templates/tracing_policy.tmpl
+TRACING_POLICY_OUTPUT := content/en/docs/reference/tracing-policy.md
+TRACING_POLICY_CRDS_DIR := ../pkg/k8s/apis/cilium.io/client/crds/v1alpha1
+
+# Path to crdoc binary (can be overridden by environment)
+# See https://github.com/fybrik/crdoc
+CRDOC ?= $(HOME)/go/bin/crdoc
```

by the way, this image is two years old :/
```suggestion
CRDOC ?= docker run --rm --platform linux/amd64 -u $(shell id -u):$(shell id -g) \
	-v $(abspath $(CURDIR)/$(TRACING_POLICY_CRDS_DIR):/crds \
	-v ${PWD}:/workdir ghcr.io/fybrik/crdoc:0.6.2
```